cmd/xcode.go

```diff
@@ -81,12 +82,23 @@ func scanXcodeProject(cmd *cobra.Command, args []string) error {
 
 	projectPath := paramXcodeProjectFilePath
 	if projectPath == "" {
-		askText := `Please drag-and-drop your Xcode Project (` + colorstring.Green(".xcodeproj") + `) or Workspace (` + colorstring.Green(".xcworkspace") + `) file, 
+
+		log.Infof("Scan the directory for project files")
+		projpth, err := scanForProjectFiles([]ProjectType{xcodeWorkspace, xcodeProject})
```

Comment:
similar notes here: https://github.com/bitrise-tools/codesigndoc/pull/96/files#r249726005

Reply:
what if we find 1 solution, but unfortunately not the one the user wants to use? the logic does not fail, so the user has no option to use another solution file?

What if:

return a list of solutions and let the user choose
setup the tool in the way that solution can be defined via flag/argument
if flag/arg passed use that solution
otherwise, scan for solution let the user choose in case of multiple solution files and also print a warning message, that specific solution file can be defined via flag/arg (print the copy-paste ready command to define the solution)